keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/GaussianVertex.java

```diff
@@ -25,6 +28,8 @@
 
     private final DoubleVertex mu;
     private final DoubleVertex sigma;
+    private final static String MU_NAME = "mu";
+    private final static String SIGMA_NAME = "sigma";
```

Comment:
can you make use of the static final `ParameterName`s in `Diffs.java`? (We should move them to another class since this is nothing to do with diffs).

Reply:
Yeah, I'll do this as part of the more general change to move over every vertex.